docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -187,10 +187,10 @@ The default bundle supports the following minimum versions:
 <!-- #stable-snapshot -->
 
 - Node 12 (up from 8)
-- Chrome 84 (up from 49)
-- Edge 85 (up from 14)
+- Chrome 90 (up from 84)
```

Comment:
This is the guide for migrating between v4 and v5. So the previous version needs to stay
```suggestion
- Chrome 90 (up from 49)
```
Same applies to the other changes

Reply:
Got it.